### DISCUSSION TITLE
Taskbar Styler ini file

### DISCUSSION THREAD

**Discussion by stedeste63** [ramensoftware/windhawk-mods#429] [Q&A]
Hi, i wrote a ps1 script to modify the aspect of a button on the taskbar when i click it. To achieve the goal i opened the ini file of Task Styler, modified it and saved it from within the script. Is there a way to load the changed ini file into Taskbar Styler(as when you "Save" from the "Advanced" tab)from the script to make the changes effective? THX in advance.

**Comment by m417z** [comment]
Hi, you also need to update `SettingsChangeTime` for it to apply the new settings.

**Reply by stedeste63** [reply]
Thank you m417z !!!
